yt_dlp/extractor/kick.py

```diff
@@ -16,21 +17,18 @@
 
 
 class KickBaseIE(InfoExtractor):
-    def _real_initialize(self):
-        self._request_webpage(
-            HEADRequest('https://kick.com/'), None, 'Setting up session', fatal=False, impersonate=True)
-        xsrf_token = self._get_cookies('https://kick.com/').get('XSRF-TOKEN')
-        if not xsrf_token:
-            self.write_debug('kick.com did not set XSRF-TOKEN cookie')
-        KickBaseIE._API_HEADERS = {
-            'Authorization': f'Bearer {xsrf_token.value}',
-            'X-XSRF-TOKEN': xsrf_token.value,
-        } if xsrf_token else {}
+    @functools.cached_property
+    def _api_headers(self):
+        token = traverse_obj(
+            self._get_cookies('https://kick.com/'),
+            ('session_token', 'value', {urllib.parse.unquote}),
+        )
```

Comment:
no hanging parentheses in function calls
```suggestion
            ('session_token', 'value', {urllib.parse.unquote}))
```